Tests/Unit/Module/STIG.PowerStigXml.tests.ps1

```diff
@@ -0,0 +1,42 @@
+#region Header
+. $PSScriptRoot\.tests.header.ps1
+#endregion
+
+$xccdfs = (Get-ChildItem -Path $script:moduleRoot\StigData\Archive -Include *xccdf.xml -Recurse | Where-Object -Property Name -Match "Server_2019_MS|IIS_10-0_Server")[1,3]
+foreach ($xccdf in $xccdfs)
+{
+    Describe "ConvertTo-PowerStigXml $($xccdf.name)" {
+
+        It 'Should return an 2 XML' {
+            ConvertTo-PowerStigXml -Path $xccdf.FullName -Destination $TestDrive -CreateOrgSettingsFile -RuleIdFilter $randomId
+            $converted = Get-ChildItem $testdrive
+            $converted.FullName.EndsWith(".xml").Count | Should be 2
+        }
+    }
+}
+
+Describe 'Compare-PowerStigXml' {
+
+    $dotNetSTIGS = (Get-ChildItem -Path $script:moduleRoot\StigData\Processed -Recurse | Where-Object -Property Name -Match "(DotNetFramework-4-.*\d.xml)").FullName
+    It 'Should return a PSObject' {
+        $Compare = Compare-PowerStigXml -OldStigPath $dotNetSTIGS[0] -NewStigPath $dotNetSTIGS[1]
```

Comment:
$Compare should begin with a lower case "c".

Reply:
Updated